packages/core/src/actions.ts

```diff
@@ -181,6 +181,8 @@ export function resolveRaise<TEvent extends EventObject>(
  * Sends an event. This returns an action that will be read by an interpreter to
  * send the event in the next step, after the current step is finished executing.
  *
+ * @deprecated Use the `sendTo(...)` action creator instead.
```

Comment:
I don't think that we can deprecate this right now. `send` is still useful to create delayed events to "self".

Reply:
This is now possible